src/tools/perf/lib/uct_tests.cc

```diff
@@ -783,21 +783,50 @@ class uct_perf_test_runner {
    TEST_CASE_ALL_OSD(_perf, _case, UCT_PERF_DATA_LAYOUT_BCOPY) \
    TEST_CASE_ALL_OSD(_perf, _case, UCT_PERF_DATA_LAYOUT_ZCOPY)
 
+#define DISPATCH(_name, _cmd, _type) \
+static ucs_status_t uct_perf_dispatch_##_name(ucx_perf_context_t *perf) { \
+    UCS_PP_FOREACH(TEST_CASE_ALL_DATA, perf, \
```

Comment:
no need to use UCS_PP_FOREACH with only one element, can invoke the macro TEST_CASE_ALL_DATA directly